python/pfs/drp/stella/readLineList.py

```diff
@@ -109,24 +119,24 @@ def getLamps(self, metadata):
         Returns
         -------
         lamps : `list` of `str`, or `None`
-            The list of lamps, if the ``restrictByLamps`` configuration option
-            is ``True`` or ``lampList`` is set; otherwise, ``None``.
+            The list of active lamps from the metadata.
+            If no active lamps can be retrieved from the metadata,
+            and ``assumeSkyIfNoLamps`` is set, then ``[skyLines]`` is returned, otherwise, ``None``.
 
         Raises
         ------
         RuntimeError
-            If ``metadata`` is ``None`` and ``restrictByLamps`` is ``True``, and
-            lampList is not set.
+            If ``metadata`` is ``None``.
         """
-        if len(self.config.lampList) > 0:
-            return self.config.lampList
         if metadata is None:
-            raise RuntimeError("Cannot determine lamps because metadata was not provided")
-        lamps = getLampElements(metadata)
+            raise RuntimeError("Cannot determine lamp information because metadata was not provided")
+        lamps = getLamps(metadata)
+        lampElementList = getLampElements(metadata)
         if not lamps and self.config.assumeSkyIfNoLamps:
             self.log.info("No lamps on; assuming sky.")
-            lamps = ["OI", "NaI", "OH"]
-        return lamps
+            lamps = ["skyLines"]
+            lampElementList = ["OI", "NaI", "OH"]
+        return lamps, lampElementList
```

Comment:
Returning a tuple isn't documented. If you're going to return more than one thing, please use a `Struct`.